indra/newview/scripts/lua/test_LLChatListener.lua

```diff
@@ -1,25 +1,49 @@
 local LLAgent = require 'LLAgent'
+local LLAppearance = require 'LLAppearance'
 local LLChat = require 'LLChat'
 local LLListener = require 'LLListener'
 local UI = require 'UI'
 
 -- Chat listener script allows to use the following commands in Nearby chat:
--- open inventory   -- open defined floater by name
--- close inventory  -- close defined floater by name
--- closeall         -- close all floaters
--- stop             -- close the script
--- any other messages will be echoed.
-function openOrEcho(message)
-    local open_floater_name = string.match(message, "^open%s+(%w+)")
-    local close_floater_name = string.match(message, "^close%s+(%w+)")
-    if open_floater_name then
-      UI.showFloater(open_floater_name)
-    elseif close_floater_name then
-      UI.hideFloater(close_floater_name)
-    elseif message == 'closeall' then
-      UI.closeAllFloaters()
+-- /open inventory   -- open defined floater by name
+-- /close inventory  -- close defined floater by name
+-- /closeall         -- close all floaters
+-- /outfit           -- wear specified outfit
+-- /help             -- show help
+-- /stop             -- close the script
+function processCommands(message)
+    print("handling message " .. message)
+    local cmd, arg = string.match(message, '^/(%w+)%s+(.*)$')
+    if arg then
+      -- remove quotes from arg
+      arg = string.gsub(arg, '"', '')
     else
-      LLChat.sendNearby('Echo: ' .. message)
+      cmd = string.match(message, '^/(%w+)')
+    end
+    print("cmd: " .. (cmd or '') .. " arg: " .. (arg or ''))
```

Comment:
This would be a good use for Luau's string interpolation (using backticks):
```
print(`cmd: {cmd} arg: {arg}`)
```
In Lua, when you're not trying to concatenate `nil` with a string, `nil` stringizes as "nil".